lib/src/locator.dart

```diff
@@ -21,6 +22,7 @@ Future initializeApp() async {
     AppInfo.initialize(),
     UserPrefs.instance.initialize(),
   ]);
+  await RemoteConfigService.getRemoteConfig();
```

Comment:
I think we should put `RemoteConfigService.getRemoteConfig()` inside `Future.wait` block.
```suggestion
    UserPrefs.instance.initialize(),
    RemoteConfigService.getRemoteConfig(),
  ]);
```